spec/fortnox/api/repositories/examples/only.rb

```diff
@@ -0,0 +1,42 @@
+
+shared_examples_for '.only' do |matching_filter, missing_filter|
+  describe '.only' do
+
+    let( :repository ){ described_class.new }
+    let( :vcr_dir ){ repository.options.json_collection_wrapper.downcase }
```

Comment:
Had to, the previous version used `subject`, but subject was declared below and they got into an infinite loop :)